### PR TITLE
Fixing some of the build issues on master

### PR DESCRIPTION
**Pull request by bergerhoffer**
Fixes these build warnings/errors on the master branch:

```
asciidoctor: ERROR: <stdin>: line 72: level 0 sections can only be used when doctype is book
asciidoctor: WARNING: modules/ossm-control-plane-web.adoc: line 6: section title out of sequence: expected level 1, got level 2
asciidoctor: WARNING: modules/ossm-control-plane-cli.adoc: line 6: section title out of sequence: expected level 1, got level 2
asciidoctor: WARNING: modules/ossm-config-sidecar-mtls.adoc: line 6: section title out of sequence: expected level 3, got level 4
asciidoctor: WARNING: modules/ossm-routing-bookinfo-applying.adoc: line 2: section title out of sequence: expected level 2, got level 3
asciidoctor: WARNING: modules/ossm-routing-bookinfo-test.adoc: line 2: section title out of sequence: expected level 2, got level 3
asciidoctor: WARNING: modules/ossm-routing-bookinfo-route.adoc: line 2: section title out of sequence: expected level 2, got level 3
```
The following warnings need to be addressed still (but I need help from original authors/contributors):
```
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 291: callout list item index: expected 12, got 13
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 291: no callout found for <12>
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 295: callout list item index: expected 13, got 14
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 296: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 285: callout list item index: expected 12, got 13
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 285: callout list item index: expected 12, got 13
asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 288: callout list item index: expected 13, got 14
asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 262: callout list item index: expected 15, got 16
asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 265: callout list item index: expected 16, got 17
asciidoctor: WARNING: modules/installation-gcp-config-yaml.adoc: line 244: callout list item index: expected 10, got 11
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 272: callout list item index: expected 11, got 12
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 292: callout list item index: expected 13, got 14
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 298: callout list item index: expected 14, got 15
asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_1.png
asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_2.png
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 182: callout list item index: expected 13, got 14
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 186: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 182: callout list item index: expected 13, got 14
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 186: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 202: no callout found for <13>
asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 216: no callout found for <14>
asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_1.png
asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_2.png
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 172: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 197: callout list item index: expected 15, got 16
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 199: callout list item index: expected 16, got 17
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 172: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 197: callout list item index: expected 15, got 16
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 199: callout list item index: expected 16, got 17
```

Previews:
* https://deploy-preview-33524--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html
* https://deploy-preview-33524--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-traffic-manage.html
* https://deploy-preview-33524--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-smcp.html

### CODE REVIEW

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -69,38 +69,38 @@ In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks.
 For more information, see xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating clusters].
 
 [id="migration-considerations"]
-= Migration considerations
+== Migration considerations
```
Comment (bergerhoffer) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:72]: @apinnick Can you please check/peer review the changes to this file. This had a second level 1 heading in the file (which you can only have 1 in a file), so it's bumping everything below it properly.

File: modules/ossm-config-sidecar-mtls.adoc [hunk 2]
```diff
@@ -3,13 +3,13 @@
 // * service_mesh/v2x/ossm-config.adoc
 
 [id="ossm-security-mtls-sidecars-incoming-services_{context}"]
-=== Configuring sidecars for incoming connections for specific services
+= Configuring sidecars for incoming connections for specific services
```
Comment (bergerhoffer) [inline modules/ossm-config-sidecar-mtls.adoc:6]: @neal-timpe or @JStickler can one of you please review the updates to these 6 ossm files? This is to get rid of warnings when building (mentioned in comment 0). It also fixes these modules to start at level 1.

**Comment by bergerhoffer** [comment]
> ```
> asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_1.png
> asciidoctor: WARNING: image to embed not found or not readable: /home/ahoffer/git/openshift-docs/installing/installing_bare_metal_ipi/images/4.4-71_OpenShift_Baremetal_IPI_Depoyment_0320_2.png
> ```

@johnwilkins Can you please take a look at these errors from the build output? I took a look at the file, but I don't see any images with these names in the repo, so I'm not sure how to resolve it.

**Comment by bergerhoffer** [comment]
> ```
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 291: callout list item index: expected 12, got 13
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 291: no callout found for <12>
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 295: callout list item index: expected 13, got 14
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 296: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 285: callout list item index: expected 12, got 13
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 285: callout list item index: expected 12, got 13
> asciidoctor: WARNING: modules/installation-aws-config-yaml.adoc: line 288: callout list item index: expected 13, got 14
> asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 262: callout list item index: expected 15, got 16
> asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 265: callout list item index: expected 16, got 17
> asciidoctor: WARNING: modules/installation-gcp-config-yaml.adoc: line 244: callout list item index: expected 10, got 11
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 272: callout list item index: expected 11, got 12
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 292: callout list item index: expected 13, got 14
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 298: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 182: callout list item index: expected 13, got 14
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 186: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 182: callout list item index: expected 13, got 14
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 186: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-bare-metal-config-yaml.adoc: line 266: callout list item index: expected 11, got 12
> asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 202: no callout found for <13>
> asciidoctor: WARNING: modules/installation-azure-config-yaml.adoc: line 216: no callout found for <14>
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 172: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 197: callout list item index: expected 15, got 16
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 199: callout list item index: expected 16, got 17
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 172: callout list item index: expected 14, got 15
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 197: callout list item index: expected 15, got 16
> asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 199: callout list item index: expected 16, got 17
> ```

@mburke5678 Can you please help take a look at these issues with callout numbering in these files? It looks like you were one of the last to work with the callouts. But there are so many conditionals that I have no idea how to fix it properly. The only one I attempted that I tracked down was for the vsphere one that the issue seemed to be with the restricted network assembly, where it jumps from callout 13 to 15 on line 172. But again, no idea how to resolve it properly.

**Comment by bergerhoffer** [comment]
Linking #33526 which is addressing the missing images issues.

**Review by JStickler — APPROVED** [review]
LGTM

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc (hunk 1 above)
Comment (apinnick) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:72]: LGTM. I must have done that by accident when I made some changes recently. Thanks for catching it.

**Comment by mburke5678** [comment]
@bergerhoffer A for https://github.com/openshift/openshift-docs/pull/33524#issuecomment-862606352, I worked on this last evening and am having some issues with the build. I am hoping to work with Vikram to resolve/troubleshoot. This might need to wait for his and my 1-1 meeting on Monday evening, if there is no hurry. I am on PTO Thurs and Fri. He is off line now, it seems, and I'm too tired (long day) to wait much longer.

**Comment by bergerhoffer** [comment]
> @bergerhoffer A for [#33524 (comment)](https://github.com/openshift/openshift-docs/pull/33524#issuecomment-862606352), I worked on this last evening and am having some issues with the build. I am hoping to work with Vikram to resolve/troubleshoot. This might need to wait for his and my 1-1 meeting on Monday evening, if there is no hurry. I am on PTO Thurs and Fri. He is off line now, it seems, and I'm too tired (long day) to wait much longer.

@mburke5678 There is definitely no hurry on this, it's just fixing the build warnings. Thanks for looking into this!

**Pull request merged**